### PR TITLE
Users can now choose a custom URL

### PR DESCRIPTION
**Pull request by eesazahed**
A user now has the option to choose a custom URL instead of a auto-generated one.

### CODE REVIEW

File: router/createUrl.js [hunk 1]
```diff
@@ -17,10 +17,18 @@ router.post('/', (req, res, next) => {
...
-      errors: ['Your custom short URL must be between 1-10 characters long.'],
-    });
+  const customURL = req.body.customURL;
+
+  if (customURL.length > 0) {
+    if (customURL.length <= 10) {
+      return res.status(400).json({
+        errors: ['Your custom short URL must be between 1-10 characters long.'],
+      });
+    }
```
Comment (TheRedstoneRadiant) [inline router/createUrl.js:27]: Shouldn't the second condition here should be reversed?

Comment (eesazahed) [reply router/createUrl.js:27]: I don't see why, because if the custom URL length is greater than 0, and less than 10, it can be accepted.

**Comment by TheRedstoneRadiant** [comment]
Awesome!

Merged :)

**Comment by eesazahed** [comment]
You need to update the replit now

**Comment by TheRedstoneRadiant** [comment]
Done, also please see #37

**Pull request merged**